services/app-api/src/handlers/postgres_migrate.ts

```diff
@@ -156,6 +157,24 @@ export const main: Handler = async (): Promise<APIGatewayProxyResultV2> => {
         return fmtMigrateError(errMsg)
     }
 
+    // Run the prisma dataMigrations.
+    // these are compiled in app-api so we can call them directly
+
+    const dataMigratorDBURL =
+        dbConnectionURL + `&connect_timeout=${connectTimeout}`
+
+    const dataMigrator = newDBMigrator(dataMigratorDBURL)
+
+    const migrationResult = await migrate(
+        dataMigrator,
+        '../dataMigrations/migrations'
```

Comment:
Oops right now the folder name looks like its `dataMigraitions` misspelled - it doesn't match this.